include/zephyr/xen/public/domctl.h

```diff
@@ -20,7 +20,7 @@
 #include "grant_table.h"
 #include "memory.h"
 
-#define XEN_DOMCTL_INTERFACE_VERSION 0x00000015
+#define XEN_DOMCTL_INTERFACE_VERSION 0x00000016
```

Comment:
Is it actually enough and no changes were made in these structures below? I have some doubts about it

Reply:
It seems enough for this particular exercise. Right change expected to come from https://github.com/xen-troops/zephyr/pull/92 (or similar)